src/Microsoft.DocAsCode.Build.Common/ExtractSearchIndex.cs

```diff
@@ -79,7 +80,20 @@ where output.Key.Equals(".html", StringComparison.OrdinalIgnoreCase)
                 }
             }
             JsonUtility.Serialize(indexDataFilePath, indexData, Formatting.Indented);
```

Comment:
`index.json` should be saved in different folder. e.g. `_site/index.json`, `_site/v1.0/index.json`, `_site/v2.0/index.json`